particles/PipeApps/MapsApp.recipes

```diff
@@ -1,13 +1,15 @@
+import '../Pipes/PipeEntity.schema'
+
 schema Address
   Text address
 
 particle RecentAddresses in './source/RecentAddresses.js'
-  in [Address] recentAddresses
+  in [PipeEntity] recentEntities
   out Address address
 
 recipe RecentAddresses
+  map 'pipe-entities' as recentEntities
```

Comment:
Does the 'pipe-entities' thing work here? Was it not working in a `create` the other day when we were looking at it together?

Ah, I see you have a comment later that maybe is about this. I couldn't find an existing issue for this syntax, can you file one with the example?

Reply:
This version of the store is created in code, where I can easily set the `id`. It's the manifest syntax that doesn't seem to work.

The example was my broken project which I worked-around, so I don't have a convenient repro. This happens a lot (aka trying to capture a repro can be time-consuming and feels off-task). But I will file a ticket with the info.